face_model/argface_model/argface_extract_features.py

```
@@ -27,61 +17,43 @@ def __init__(self, data_path):
             if not os.path.isdir(person_path):
-                logger.warning(f"Skipping {person_path} as it is not a directory")
                 continue
             for image_name in os.listdir(person_path):
                 image_path = os.path.join(person_path, image_name)
                 if not image_path.lower().endswith(('.png', '.jpg', '.jpeg', '.bmp')):
-                    logger.warning(f"Skipping {image_path} as it is not an image file")
                     continue
-                logger.info(f"Opening image: {image_path}")
                 try:
                     image = Image.open(image_path).convert("RGB")
                 except Exception as e:
```

Ruff

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)